src/index.html

```diff
@@ -31,9 +33,140 @@
     </nav>
   </header>
 
-  <main></main>
+  <keyvisual class="p-main">
+    <div class="p-main__photo-wrapper">
+      <img class="p-main__photo-wrapper--img" src="./img/picture/main.png" alt="メインビジュアル">
```

Comment:
[imo]
alt属性は空で良さそうです！

```suggestion
      <img class="p-main__photo-wrapper--img" src="./img/picture/main.png" alt="">
```

画像自体に特に意味がない時は、無理にalt属性に値を設定する必要はないです！
この画像は背景画像としての役割だけで、画像がなかったとしても文章構造上は問題ないはずなので、alt属性は空で問題ないと思います！　

Reply:
空に修正しました！